src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/PaymentMessageHandler.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.handler;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.client.payhub.PayHubClientException;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentMessage;
+
+
+@Service
+public class PaymentMessageHandler {
```

Comment:
Either this class can be introduced by @aliveni as she is working on it or we can throw NotImplemeted exception in the below method.

Reply:
added